src/test/ui/nll/relate_tys/hr-fn-aau-eq-abu.rs

```diff
@@ -16,9 +16,7 @@
 // another -- effectively, the single lifetime `'a` is just inferred
```

Comment:
Github will not let me leave a comment directly on the relevant line, because it is not part of the diff here, but starting on line 11 there is a parenthetical that says "(though the current code actually gets it wrong, see below)"; you should remove that parenthetical as part of this commit.